pkgs/applications/radio/sdrangel/default.nix

```diff
@@ -0,0 +1,91 @@
+{stdenv, mkDerivation,fetchFromGitHub,
+cmake, gcc, pkgconfig,glew,opencv3,libusb,libopus,
+boost,qtbase,qtwebsockets,qtmultimedia,wrapQtAppsHook,
+fftw,fftwFloat,fftwLongDouble,fftwSinglePrec,
+limesuite,
+ocl-icd,
+pulseaudioSupport ? true,libpulseaudio,libav,libiio
+}:
+
+assert pulseaudioSupport -> libpulseaudio != null;
+
+let
+  qtlibs = [
+    qtbase qtwebsockets qtmultimedia wrapQtAppsHook
+  ];
+  fftwlibs = [
+    fftw fftwFloat fftwLongDouble fftwSinglePrec
+  ];
+
+  codec2 = stdenv.mkDerivation {
+    name = "codec2";
+    src = fetchFromGitHub {
+      owner = "drowe67";
+      repo = "codec2";
+      rev = "567346818c0d4d697773cf66d925fdb031e15668";
+      sha256 = "0ngqlh2cw5grx2lg7xj8baz6p55gfhq4caggxkb4pxlg817pwbpa";
+    };
+    buildInputs = [ cmake ];
+  };
+
+  cm256cc = stdenv.mkDerivation {
+    name = "cm256cc";
+    src = fetchFromGitHub {
+      owner = "f4exb";
+      repo = "cm256cc";
+      rev = "f21e8bc1e9afdb0b28672743dcec111aec1d32d9";
+      sha256 = "1imxg9hxh92r8ip161980axavvx1sr321lw0g8rbza59s50sd17g";
+    };
+    buildInputs = [ cmake boost ];
+  };
+
+  serialdv = stdenv.mkDerivation {
+    name = "serialdv";
+    src = fetchFromGitHub {
+      owner = "f4exb";
+      repo = "serialdv";
+      rev = "d5830fae715f4acffa2acec33539c3a11c17a1c9";
+      sha256 = "0swalyp8cbs7if6gxbcl7wf83ml8ch3k7ww4hws89rzpjvf070fr";
+    };
+    buildInputs = [ cmake ];
+  };
+
+  fetchedDeps = [ codec2 cm256cc serialdv ];
+in
+  mkDerivation rec {
+    name = "sdrangel-${version}";
+    version = "4.11.6";
+    src = fetchFromGitHub {
+      owner = "f4exb";
+      repo = "sdrangel";
+      rev = "ec22056cbf43f1150c610fd14df447cdadeadd45";
+      sha256 = "05vijflnydl10l0h7ib6pwi06fmg6k740vcbkm0g5293s2z85j14";
+      fetchSubmodules = false;
+    };
+    buildInputs = [
+      cmake gcc pkgconfig glew opencv3 libusb boost libopus limesuite libav libiio
+    ]
+    ++qtlibs
```

Comment:
I would not group inputs in that case. You can just add all inputs directly here.